test/apps/print_trigger_type.cxx

```diff
@@ -0,0 +1,39 @@
+/**
+ * @file check_fragment_TPs.cxx Read TP fragments from file and check that they have start times within the request window
+ *
+ * This is part of the DUNE DAQ Application Framework, copyright 2020.
+ * Licensing/copyright details are in the COPYING file that you should have
+ * received with this code.
+ */
+#include "CLI/CLI.hpp"
+
+#include "detdataformats/trigger/TriggerPrimitive.hpp"
+#include "hdf5libs/HDF5RawDataFile.hpp"
+
+#include <daqdataformats/Fragment.hpp>
+#include <daqdataformats/FragmentHeader.hpp>
+#include <daqdataformats/TriggerRecordHeader.hpp>
+#include <daqdataformats/Types.hpp>
+#include <iostream>
+
+int
+main(int argc, char** argv)
+{
+  CLI::App app{ "App description" };
+
+  std::string filename;
+  app.add_option("-f,--file", filename, "Input HDF5 file");
+
+  CLI11_PARSE(app, argc, argv);
+
+  dunedaq::hdf5libs::HDF5RawDataFile decoder(filename);
+
+  auto trigger_record_numbers = decoder.get_all_trigger_record_numbers();
+  
+  // Populate the map with the TRHs and DS fragments
```

Comment:
Oops I forgot to remove this now-obsolete comment. Could you remove it please?

Reply:
removed